README.md

```diff
@@ -15,6 +15,11 @@ If you wish to install the development version from GitHub (which often contains
 remotes::install_github("rstudio/rticles")
 ```
 
+If you want to install a locally modified version, clone this repo and do:
+```r
+remotes::install_local("path/to/local/repo")
+```
+
```

Comment:
This is not specific to **rticles** - this is development practice. You can also use `install.packages` or RStudio IDE included  build & install feature. 

I don't think we should add that here. Why do you suggest that ?

Reply:
I think it's good to leave a note on how to modify and install a local development version. This would make it easier for people to contribute. Maybe move this under "Development" or "Contribute" section?

Reply:
It we keep something like that it should be moved to contribute section yes. I would prefer to add a link to some documentation about how to work and develop r package like https://r-pkgs.org/index.html instead of developing  a new part on how to develop.

Reply:
I'd also prefer leaving this out. As @cderv said, this is development practice and not specific to the **rticles** package. There are several ways to install a local package, and `remotes::install_local()` is one of them. RStudio users will probably find `Ctrl/Cmd + Shift + B` easier.